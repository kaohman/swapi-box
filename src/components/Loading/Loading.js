import React from 'react';

const Loading = () => {
  return(
    <div className='no-cards-container'>
      <img className='star-wars-images' src={require('../../images/darth-vader.png')} alt='darth vader' />
      <h3 className='no-cards-text'>Loading data... I find your lack of faith disturbing</h3>
      <div className="lds-roller"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
    </div>
  )
}

export default Loading;