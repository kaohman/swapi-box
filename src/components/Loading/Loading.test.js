import React from 'react';
import ReactDOM from 'react-dom';
import Loading from './Loading';
import { shallow } from 'enzyme';

describe('Loading', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Loading />
    );
  });

  it('should match the snapshot with all data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should receive the text, title, and date for a movie given as props from app', () => {
    // expect(wrapper.props()).toEqual('this is sample text');
  });
});