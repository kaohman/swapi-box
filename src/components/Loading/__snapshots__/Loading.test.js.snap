// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot with all data passed in 1`] = `
<div
  className="no-cards-container"
>
  <img
    alt="darth vader"
    className="star-wars-images"
    src="darth-vader.png"
  />
  <h3
    className="no-cards-text"
  >
    Loading data... I find your lack of faith disturbing.
  </h3>
  <div
    className="lds-roller"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
